src/pages/git/git-branch/index.md

```diff
@@ -33,15 +33,15 @@ To checkout an existing branch, run the command:
 git checkout BRANCH-NAME
 ```
 
-Generally, Git won't let you checkout another branch unless your working directory is clean, because you would lose any working directory changes that aren't committed. You have three options to handle your changes: 1) trash them (see [Git checkout for details](../git-checkout/index.md)), 2) commit them (see [Git commit for details](../git-commit/index.md)), or 3) stash them (see [Git stash for details](../git-stash/index.md)).
+Generally, Git won't let you checkout another branch unless your working directory is clean, because you would lose any working directory changes that aren't committed. You have three options to handle your changes: 1) trash them (see [Git checkout for details](https://guide.freecodecamp.org/git/git-checkout/)), 2) commit them (see [Git commit for details](https://guide.freecodecamp.org/git/git-commit/)), or 3) stash them (see [Git stash for details](https://guide.freecodecamp.org/git/git-stash/)).
 
 ### Create a New Branch
 To create a new branch, run the command:
 ```shell
 git branch NEW-BRANCH-NAME
 ```
 
-Note that this command only creates the new branch. You'll need to run `git checkout <newBranchName>` to switch to it.
+Note that this command only creates the new branch. You'll need to run `git checkout NEW-BRANCH-NAME` to switch to it.
```

Comment:
Is all-caps the style guide for fcc Guides? I ask because I often see the `<insert your name>` style formatting in coding tutorials and thought it was fairly acceptable.

Reply:
@dhcodes - we had an issue a while back where the angle brackets were being rendered as HTML, even when they were in code blocks. I believe that issue is now fixed, but as a result, we switched over to the all caps with hyphens formatting just to be safe.